scalafix-outputs/src/main/scala/fix/Http4s018To020.scala

```diff
@@ -53,8 +53,8 @@ object Http4s018To020 {
     group = None
   )
 
-  val client: cats.effect.Resource[IO, Client[IO]] = BlazeClientBuilder[IO](ExecutionContext.fromExecutorService(Executors.newFixedThreadPool(1))).resource
+  val client: IO[Client[IO]] = BlazeClientBuilder[IO](ExecutionContext.fromExecutorService(Executors.newFixedThreadPool(1))).allocate.map{ case (client, shutdown) => client }
```

Comment:
would this be ok? I'm not sure what to do with the second value if we want to keep the same type

Reply:
We would not be able to shut it down with that. I think we need to return the IO[Client[IO], F[Unit]]